docs/docs/guides/application-development.md

```diff
@@ -80,6 +80,18 @@ dm -u http://localhost:5000 reset app
 
 This must be done every time a modification is done in any file related to a service.
 
+## UI recipes
+
+The UI recipe links are located under `app/data/DemoApplicationDataSource/instances/recipe_links/`. 
+
+To create lookup table for UI recipes, run:
+
+```
+dm create-lookup demo-app DemoApplicationDataSource/instances/recipe_links
+```
+
+This must be done every time a modification is done in any recipe link files.
+
```

Comment:
```suggestion
## UiPlugins and UiRecipes

### Adding a UiPlugin
1. Create a folder in `src/plugins`. This will be your plugin, and must have a `index.js`-file that has a default export of the type `export const plugins: TPlugin[]`
2. Add this plugin to the list of plugins to load in `src/plugins.js`
3. Done!

### Using the UiPlugin

Associate a blueprint type with a set of _UiRecipes_ that uses the UiPlugin

1. Create a `CORE:RecipeLink`-entity (see `app/data/DemoApplicationDataSource/instances/recipe_links/demoApp.json` for an example)
2. Create a _lookup_ with the app name and the _RecipeLinks_ for it (DMSS must be running on localhost:5000)
   - `dm create-lookup demo-app DemoApplicationDataSource/instances/recipe_links`
```